src/stores/Annotation/Annotation.js

```diff
@@ -1038,7 +1038,14 @@ export const Annotation = types
         self.acceptAllSuggestions();
       } else {
         self.suggestions.forEach((suggestion) => {
-          if (['richtextregion', 'text', 'textrange'].includes(suggestion.type)) {
+          // regions that can't be accepted in usual way, should be auto-accepted;
+          // textarea will have simple classification area with no type, so check result.
+          // @todo per-regions is tough thing here as they can be in generated result,
+          // connected to manual region, will check it later
+          const onlyAutoAccept = ['richtextregion', 'text', 'textrange'].includes(suggestion.type)
```

Comment:
not a show stopper by any means but to help describe what we're checking naming this array would make this code a little more self documenting
```suggestion
          const textTypes = ['richtextregion', 'text', 'textrange'];
          const onlyAutoAccept = textTypes.includes(suggestion.type)
```